fdbserver/DataDistribution.actor.cpp

```diff
@@ -4244,25 +4301,31 @@ ACTOR Future<Void> clusterHealthCheckForPerpetualWiggle(DDTeamCollection* self,
 		wait(delay(SERVER_KNOBS->CHECK_TEAM_DELAY, TaskPriority::DataDistributionLow));
 	}
 }
-// Watches the value (pid) change of \xff/storageWigglePID, and adds storage servers held on process of which the
-// Process Id is “pid” into excludeServers which prevent recruiting the wiggling storage servers and let teamTracker
-// start to move data off the affected teams. The wiggling process of current storage servers will be paused if the
-// cluster is unhealthy and restarted once the cluster is healthy again.
+// Watches the value change of `wigglingStorageServerPrefix`, and adds the storage server into excludeServers which
+// prevent recruiting the wiggling storage servers and let teamTracker start to move data off the affected teams. The
+// wiggling process of current storage servers will be paused if the cluster is unhealthy and restarted once the cluster
+// is healthy again.
 ACTOR Future<Void> perpetualStorageWiggler(AsyncVar<bool>* stopSignal,
                                            PromiseStream<Void> finishStorageWiggleSignal,
                                            DDTeamCollection* self) {
 	state Future<Void> watchFuture = Never();
 	state Future<Void> moveFinishFuture = Never();
 	state int extraTeamCount = 0;
 	state Future<Void> ddQueueCheck = clusterHealthCheckForPerpetualWiggle(self, &extraTeamCount);
-	state int movingCount = 0;
-	state std::pair<Future<Void>, Value> res = wait(watchPerpetualStoragePIDChange(self));
-	ASSERT(!self->wigglingPid.present()); // only single process wiggle is allowed
-	self->wigglingPid = Optional<Key>(res.second);
+	state std::pair<Future<Void>, Optional<Value>> res = wait(watchPerpetualStorageIDChange(self));
+	ASSERT(!self->wigglingId.present()); // only single process wiggle is allowed
```

Comment:
I would put a "SOMEDAY OR TODO" as in future we would like to extend the algo to wiggle multiple processes if needed...